templates/includes/pagination.html

```diff
@@ -0,0 +1,15 @@
+<ul class="pagination">
+  {% if post_list.has_previous %}
+    <li class="page-item">
+      <a href="?page={{ post_list.previous_page_number }}" class="page-link">prev</a>
```

Comment:
The 'prev' link should be disabled or hidden when there is no previous page. Consider adding a condition to check if 'post_list.has_previous' is true before rendering this link.